apps/admin/frontend/src/api.ts

```diff
@@ -231,6 +231,25 @@ export const getWriteInAdjudicationQueue = {
   },
 } as const;
 
+type GetFirstPendingWriteInIdInput = QueryInput<'getFirstPendingWriteInId'>;
+export const getFirstPendingWriteInId = {
+  queryKey(input?: GetFirstPendingWriteInIdInput): QueryKey {
+    return input
+      ? ['getFirstPendingWriteInId', input]
+      : ['getFirstPendingWriteInId'];
+  },
+  useQuery(input: GetFirstPendingWriteInIdInput) {
+    const apiClient = useApiClient();
+    return useQuery(
+      this.queryKey(input),
+      () => apiClient.getFirstPendingWriteInId(input),
+      {
+        cacheTime: 0,
```

Comment:
I want the query to re-run anytime its mounted and never rely on cached queries.